vtex/styleguide
# Fix/slider classes

#### What is the purpose of this pull request?

add a new css class based on vtex styles@2.x rules



#### What problem is this solving?

now it's possible to style this element with native css handle and without deprecated css selectors



#### How should this be manually tested?

Go to any VTEX IO store, make a search for products and Inspect Filter Navigator component with DevTools, you will could see the element inside namespace vtex-search-result-3-x-filter__container--priceRange without vtex namespace.

Or you can go on VTEX Styleguide look for Slider component and inspect with DevTools, you will see that slider range values are without css namespaces.

#### Screenshots or example usage

Element without css namespaces -> https://prnt.sc/sg6snq
Element with css namespaces added -> https://prnt.sc/sg6u2j

#### Types of changes
